.github/workflows/capabilities.yml

```diff
@@ -0,0 +1,33 @@
+name: '@web3-storage/capabilities'
+env:
+  CI: true
+  FORCE_COLOR: 1
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'packages/capabilities/**'
+      - '.github/workflows/capabilities.yml'
+      - 'pnpm-lock.yaml'
+  pull_request:
+    paths:
+      - 'packages/capabilities/**'
+      - '.github/workflows/capabilities.yml'
+      - 'pnpm-lock.yaml'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.3
+        with:
+          version: 7
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'pnpm'
+      - run: pnpm install
+      - run: pnpm -r --filter @web3-storage/capabilities run build
```

Comment:
I don't believe we need to run the build, it mostly does typecheck which is better handled by separate job as suggested in the comment above.

Reply:
removed it when adding typecheck